policyengine_us/variables/gov/states/mt/tax/income/mt_capital_gains_tax_joint.py

```diff
@@ -0,0 +1,56 @@
+from policyengine_us.model_api import *
+
+
+class mt_capital_gains_tax_joint(Variable):
+    value_type = float
+    entity = Person
+    label = "Montana net long-term capital gains tax when married couples file jointly"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://mtrevenue.gov/wp-content/uploads/dlm_uploads/2023/12/Form_2_2023_Instructions.pdf#page=6"  # Net Long-Term Capital Gains Tax Table
+    defined_for = StateCode.MT
+
+    def formula(person, period, parameters):
+        p = parameters(period).gov.states.mt.tax.income.main.capital_gains
+        # the tax for capital gains comes into effect after 2024
+        if not p.availability:
```

Comment:
include the calculation in the main part of the if statement - reduces computation stain on years pre 2024

Reply:
Please let me know if the modified codes work.